language-server/modules/langserver-core/src/test/resources/completion/annotation_ctx/config/paramAnnotation4.json

```diff
@@ -6,49 +6,178 @@
   "source": "annotation_ctx/source/paramAnnotation4.bal",
```

Comment:
Shall we verify this change? For this particular case, we should not get the types. Only the annotations should get

Reply:
https://github.com/ballerina-platform/ballerina-lang/blob/master/language-server/modules/langserver-core/src/test/resources/completion/annotation_ctx/source/paramAnnotation4.bal
This is the failing test case. In line 12, cursor is at `function name(@module1: <cursor>`. With this change it is being routed to `RequiredParameterNodeContext` instead of `AnnotationNodeContext`. I tried with a few type definitions, and I'm not getting their completions at cursor position. However, `AnnotationType` in the example is seen to be suggested. Should it be avoided?

Reply:
This is an issue with the test case as I understood. Can you change the character value to 23?. That should work.

Reply:
Done!